src/components/Pivot/Pivot.md

```diff
@@ -0,0 +1,30 @@
+### Notes for use
+
+Add here any notes on how to use the component, or delete this section.
```

Comment:
you can probably just delete the "Notes for use" section, it's pretty straightforward